test/errors/tupleExpandNonTuple-detailed.good

```diff
@@ -0,0 +1,9 @@
+─── error in tupleExpandNonTuple.chpl:3 [TupleExpansionNonTuple] ───
+  Cannot apply tuple expansion to non-tuple argument.
+  In the following function call:
+      |
+    3 | f((...x));
+      |   ⎺⎺⎺⎺⎺⎺   
+      |
+  the expanded element has type 'int(64)' while it's expected to be a tuple.
```

Comment:
I'd use "where" rather than "while" to avoid any potential implications of concurrent behavior/timing.  Or perhaps something like "has non-tuple type 'int(64)', yet tuple expansion can only be performed on tuple expressions."